aws_sagemaker_studio/frameworks/tensorflow_mnist/tensorflow_mnist.ipynb

```diff
@@ -137,11 +137,10 @@
     "\n",
     "estimator = TensorFlow(entry_point=training_script,\n",
     "                       role=role,\n",
-    "                       train_instance_count=1,\n",
-    "                       train_instance_type='ml.p2.xlarge',\n",
+    "                       instance_count=1,\n",
+    "                       instance_type='ml.p2.xlarge',\n",
     "                       framework_version=tf_version,\n",
-    "                       py_version='py3',\n",
-    "                       distributions={'parameter_server': {'enabled': True}})"
```

Comment:
Makes sense that you don't need this with one node, but was this breaking something?
Can you put in the PR description which issue this fixes?

Reply:
I'm not aware of any specific issues in GitHub that this fixes.